x-pack/legacy/plugins/ml/server/routes/job_service.ts

```diff
@@ -0,0 +1,609 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { schema } from '@kbn/config-schema';
+import { licensePreRoutingFactory } from '../new_platform/licence_check_pre_routing_factory';
+import { wrapError } from '../client/error_wrapper';
+import { RouteInitialization } from '../new_platform/plugin';
+import {
+  catFieldExamplesSchema,
+  chartSchema,
+  datafeedIdsSchema,
+  forceStartDatafeedSchema,
+  jobIdsSchema,
+  jobsWithTimerangeSchema,
+  lookBackProgressSchema,
+  topCategoriesSchema,
+  updateGroupsSchema,
+} from '../new_platform/job_service_schema';
+// @ts-ignore no declaration module
+import { jobServiceProvider } from '../models/job_service';
```

Comment:
would be great to migrate this file to TS, because it's related to job service routes 🙂

Reply:
Good idea - I feel that might be a lot for this PR so I'd like to address it in a follow-up. 👌